mb-rest-api.php

```diff
@@ -54,6 +61,38 @@ public function get_post_meta( $object ) {
 
 		return $output;
 	}
+	
+	/**
+	 * Update post meta for the rest API.
+	 *
+	 * @param json string $value post_custom, array $object Post object
+	 *
+	 * @return array
+	 */
+	public function update_post_meta_rest_api( $value, $object ) {
+
+		function get_json($string) {
+			$json = json_decode($string, true);
+			if(json_last_error() != JSON_ERROR_NONE){
+				return false;
+			}
+			return $json;
+		}
```

Comment:
Thanks for your contribution! Can you remove this `get_json` function as it's a closure and might not work on PHP 5.2. Maybe we just use `@json_decode`?